### PR TITLE
new flame_wave inputs

### PR DESCRIPTION
**Pull request by zingale**
## PR summary

this are the inputs I am using (so far) for the 30 cm H/He burn

## PR checklist

- [ ] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

File: Exec/science/flame_wave/inputs_H_He/inputs.H_He.1000Hz [hunk 1]
```diff
@@ -60,7 +61,7 @@ castro.react_T_min = 6.e7
 castro.cfl            = 0.8     # cfl number for hyperbolic system
 castro.init_shrink    = 0.1     # scale back initial timestep
 castro.change_max     = 1.1     # max time step growth
-
+castro.dt_cutoff      = 5.e-20  # level 0 timestep below which we halt
```
Comment (maxpkatz) [inline Exec/science/flame_wave/inputs_H_He/inputs.H_He.1000Hz:64]: Let's keep this one out

File: Exec/science/flame_wave/inputs_boost/inputs.boost_10_10.3d [hunk 2]
```diff
@@ -56,7 +61,7 @@ castro.react_T_min = 6.e7
 castro.cfl            = 0.8     # cfl number for hyperbolic system
 castro.init_shrink    = 0.1     # scale back initial timestep
 castro.change_max     = 1.1     # max time step growth
-
+castro.dt_cutoff      = 5.e-20  # level 0 timestep below which we halt
```
Comment (maxpkatz) [inline Exec/science/flame_wave/inputs_boost/inputs.boost_10_10.3d:64]: Same comment

File: Exec/science/flame_wave/inputs_H_He/inputs.H_He.1000Hz (hunk 1 above)
Comment (zingale) [reply Exec/science/flame_wave/inputs_H_He/inputs.H_He.1000Hz]: I removed this

**Comment by zingale** [comment]
I also changed the H/He inputs to use 30 cm resolution and fixed the BCs in the boost inputs.

**Pull request merged**